rpcs3/rpcs3qt/log_frame.cpp

```diff
@@ -216,21 +236,53 @@ void log_frame::CreateAndConnectActions()
 		connect(act, &QAction::triggered, [this, logLevel]()
 		{
 			s_gui_listener.enabled = std::max(logLevel, logs::level::fatal);
-			xgui_settings->SetValue(gui::l_level, static_cast<uint>(logLevel));
+			m_gui_settings->SetValue(gui::l_level, static_cast<uint>(logLevel));
 		});
 	};
 
 	m_clearAct = new QAction(tr("Clear"), this);
-	connect(m_clearAct, &QAction::triggered, m_log, &QTextEdit::clear);
+
+	if (m_plain_log)
+	{
+		connect(m_clearAct, &QAction::triggered, m_plain_log, &QPlainTextEdit::clear);
+		connect(m_plain_log, &QWidget::customContextMenuRequested, [=](const QPoint& pos)
+		{
+			QMenu* menu = m_plain_log->createStandardContextMenu();
+			menu->addAction(m_clearAct);
+			menu->addSeparator();
+			menu->addActions(m_logLevels->actions());
+			menu->addSeparator();
+			menu->addAction(m_stackAct_log);
+			menu->addSeparator();
+			menu->addAction(m_TTYAct);
+			menu->exec(m_plain_log->viewport()->mapToGlobal(pos));
+		});
+	}
+	else if (m_log)
+	{
+		connect(m_clearAct, &QAction::triggered, m_log, &QTextEdit::clear);
+		connect(m_log, &QWidget::customContextMenuRequested, [=](const QPoint& pos)
+		{
+			QMenu* menu = m_log->createStandardContextMenu();
+			menu->addAction(m_clearAct);
+			menu->addSeparator();
+			menu->addActions(m_logLevels->actions());
+			menu->addSeparator();
+			menu->addAction(m_stackAct_log);
+			menu->addSeparator();
+			menu->addAction(m_TTYAct);
+			menu->exec(m_log->viewport()->mapToGlobal(pos));
+		});
+	}
 
 	m_clearTTYAct = new QAction(tr("Clear"), this);
 	connect(m_clearTTYAct, &QAction::triggered, m_tty, &QTextEdit::clear);
 
 	m_stackAct_tty = new QAction(tr("Stack Mode (TTY)"), this);
 	m_stackAct_tty->setCheckable(true);
-	connect(m_stackAct_tty, &QAction::toggled, xgui_settings.get(), [=](bool checked)
+	connect(m_stackAct_tty, &QAction::toggled, m_gui_settings.get(), [=](bool checked)
```

Comment:
```suggestion
	connect(m_stackAct_tty, &QAction::toggled, [=](bool checked)
```

Passing this pointer is redundant since you're connecting a lambda, yeah? I think that's the case.

Reply:
Jesus. Stick to the topic